src/index.js

```diff
@@ -1,4 +1,5 @@
 import Client from './Client';
+import Authenticator from './Authenticator';
 
 // eslint-disable-next-line import/prefer-default-export
-export { Client };
+export { Client, Authenticator };
```

Comment:
Now you can remove the upper line and the linter won`t complain. Since now you are exporting two classes

Reply:
Done. Thanks!